testdroid_cmdline.sh

```diff
@@ -689,6 +734,9 @@ done
 
 cd $zip_temp_dir || exit 34
 
+# prevent sending old results to TD
+rm -rf results
```

Comment:
Consider adding `results` to `CLEANUP_FILES`-variable instead:
https://github.com/Applifier/testdroid-ssa-client/blob/feature/testdroid-client-timeout/testdroid_cmdline.sh#L18